ingestors/kafka/src/main/scala/hydra/kafka/model/TopicMetadataV2Request.scala

```diff
@@ -15,23 +19,42 @@ case object RestrictedEmployeeData extends DataClassification
 
 sealed trait ContactMethod
 
-final case class Email(address: String) extends ContactMethod
-final case class Slack(channel: String) extends ContactMethod
+object ContactMethod {
```

Comment:
Create Tests, possibly new small Spec, to test the apply functionality of the contact methods.

Reply:
Created in 569de59